packages/core/stories/text/text.doc.stories.mdx

```diff
@@ -32,22 +32,21 @@ We've created a base Text component (that contains our key typography styles and
 
 You can also create your own text components using the props and CSS variables to configure them.
 
-# API
+# Features
 
-```
-import { Text, H1, H2, H3, H4, Label, Display1, Display2, Display3 } from "@jpmorganchase/uitk-core";
-```
+## Highlighting
 
-## Props
+Text provides a default highlight color. Use the token `text-background-selected` to indicate a selection of text using a cursor, in both light and dark modes. 
 
-<ArgsTable of={Text} />
+A selection of text should be able to be highlighted with the default highlight color applied, this includes text within input fields.
 
-## Features
+The highlight function can also be used to highlight images, or other content, alongside text. 
 
```

Comment:
I don't think we need to mention this here

Reply:
OK